packages/hardhat-core/src/internal/cli/analytics.ts

```diff
@@ -19,60 +16,63 @@ import { getPackageJson } from "../util/packageInfo";
 
 const log = debug("hardhat:core:analytics");
 
-// VERY IMPORTANT:
-// The documentation doesn't say so, but the user-agent parameter is required (ua).
-// If you don't send it, you won't get an error or anything, Google will *silently* drop your hit.
-//
-// https://stackoverflow.com/questions/27357954/google-analytics-measurement-protocol-not-working
-interface RawAnalytics {
-  v: "1";
-  tid: string;
-  cid: string;
-  dp: string;
-  dh: string;
-  t: string;
-  ua: string;
-  cs: string;
-  cm: string;
-  cd1: string;
-  cd2: string;
-  cd3: string;
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
I think this is better than multiple `disable-next-line`, but feel free to undo if you disagree.